daq/gcp.py

```diff
@@ -5,14 +5,20 @@
 import os
 import sys
 
-import firebase_admin
-from firebase_admin import credentials
-from firebase_admin import firestore
-
-from google.cloud import pubsub_v1
-from google.cloud import storage
-from google.cloud import logging
-from google.auth import _default as google_auth
+SUCCESSFUL_IMPORTS = True
```

Comment:
Move SUCCESSFUL_IMPORTS to the end of the try: block (again, non-mutated variables)